docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+  ❌ Ory Tunnel doesn't convert sessions into JSON Web Tokens, doesn't issue JTWs, 
+  and does't use JWTs natively <br />
+  ❌ Add the `.ory` prefix to any of the Ory URLs
+
+To run the Tunnel, use the `ory tunnel` command and point to the URL with your application, for example: 
+
+```shell
+ory tunnel http://localhost:3000
+```
+
+### Usage
+
+You can use Ory Tunnel the same way you can use Ory Proxy. The tool can be used for local development, 
+in production, on different server types, and in Docker. 
+
+For more details, refer to the [Ory Proxy documentation](#ory-proxy).
+
+## URL Structure
+
+:::tip
+Ory Proxy adds `/.ory` prefix when mirroring the Ory Cloud APIs and UIs. 
+:::
+
+For example, when using the Ory Proxy, calling 
+`https://<proxy-host>/.ory/ui/login` is the same as calling `https://<your-project-slug>.projects.oryapis.com/ui/login`
+directly.
+
+### Self-service Flows
+
+Ory Proxy and Tunnel provide URLs which you can use to initiate self-service flows:
+
+````mdx-code-block
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'
+
+<Tabs groupId="proxy-tunnel">
+  <TabItem value="proxy" label="Ory Proxy">
+
+  | URL   |  Self-service Flow  |
+  |:--- |:--- |
+  | `/.ory/self-service/login/browser`| [Login](/kratos/self-service/flows/user-login)  |
+  | `/.ory/self-service/registration/browser`| [Registration](/kratos/self-service/flows/user-registration) |
+  | `/.ory/self-service/recovery/browser`| [Recovery](/kratos/self-service/flows/account-recovery)  |
+  | `/.ory/self-service/verification/browser`| [Verification](/kratos/self-service/flows/verify-email-account-activation) |
+  | `/.ory/self-service/settings/browser`| [Settings](/kratos/self-service/flows/user-settings) |
+
+  </TabItem>
+  <TabItem value="tunnel" label="Ory Tunnel">
+
+  | URL   |  Self-service Flow  |
+  |:--- |:--- |
+  | `/self-service/login/browser`| [Login](/kratos/self-service/flows/user-login)  |
+  | `/self-service/registration/browser`| [Registration](/kratos/self-service/flows/user-registration) |
+  | `/self-service/recovery/browser`| [Recovery](/kratos/self-service/flows/account-recovery)  |
+  | `/self-service/verification/browser`| [Verification](/kratos/self-service/flows/verify-email-account-activation) |
+  | `/self-service/settings/browser`| [Settings](/kratos/self-service/flows/user-settings) |
+
+  </TabItem>
+</Tabs>
+````
+
+If you redirect or link the user to one of those paths, the corresponding 
+self-service flow is initiated, for example:
+
+````mdx-code-block
+<Tabs groupId="proxy-tunnel">
+  <TabItem value="proxy" label="Ory Proxy">
+
+```html
+<a href="/.ory/self-service/login/browser">Log in</a>
+```
+
+  </TabItem>
+  <TabItem value="tunnel" label="Ory Tunnel">
+
+```html
+<a href="/self-service/login/browser">Log in</a>
+```
+
+  </TabItem>
+</Tabs>
+````
+
+You can also append a `return_to` query parameter to the URL. This redirects the user
+to the set URL when they complete the self-service flow.
```

Comment:
```suggestion
You can also append a `return_to=<your-url>` query parameter to the URL. This redirects the user
to the set URL when they complete the self-service flow.
```

maybe add this so its clear what you mean by "set URL"?

Reply:
done